README.markdown

```diff
@@ -1079,6 +1080,31 @@ how the result will be used. Below is a diagram showing the order in which direc
 
 [Back to TOC](#table-of-contents)
 
+lua_load_resty_core
+-------------------
+
+**syntax:** *lua_load_resty_core on|off*
+
+**default:** *lua_load_resty_core on*
+
+**context:** *http*
+
+Controls whether the `resty.core` module (from [lua-resty-core](https://github.com/openresty/lua-resty-core))
+should be loaded or not. When enabled, this directive is equivalent to executing the following when the Lua VM is
+created:
+
+```lua
+
+ require "resty.core"
+```
+
+Note that usage of the `resty.core` module is recommended, as its FFI implementation is both faster and
+more complete than the Lua C API of the ngx_lua module.
```

Comment:
And also "safer". Thinking about the "dead lock" risk in the shdict API based on CFunction.

Also, we can add a caveat that we will remove the old CFunction-based Lua API implementation in the very near future and the use of lua-resty-core will be mandated.

Reply:
Right, I added both notes to the documentation.